orchestra/proc-macro/src/impl_subsystem_ctx_sender.rs

```diff
@@ -282,7 +282,19 @@ pub(crate) fn impl_subsystem_sender(
 						self.signals_received.load(),
 						<#all_messages_wrapper as ::std::convert::From<_>> ::from (
 							<#outgoing_wrapper as ::std::convert::From<_>> :: from ( msg )
-						)
+						),
+						ChannelsOutPriority::Normal,
+					).await;
+				}
+
+				async fn priority_send_message(&mut self, msg: OutgoingMessage)
```

Comment:
If we use the generic with default value, we could avoid this additional public API

Reply:
Hmm, in that case I propose a slighly different API `send_message` and `send_message_with_priority<Priority>` which is called by `send_message`.